CHANGES.rst

```diff
@@ -6,9 +6,11 @@ chronological order. We follow `semantic versioning <https://semver.org/>`_ and
 releases are available on `Anaconda.org <https://anaconda.org/gettsim/gettsim>`_.
 
 
-0.x.x - 2020-06-xx
+0.3.2 - 2020-06-xx
```

Comment:
Probably this will go into 0.4 as it will change the interface of `compute_taxes_and_transfers`, IIUC?

Reply:
The visualization does not change anything with `compute_taxes_and_transfers`. If you mean separating the dag from the interface, then, yes, but I would like to tackle this in a different PR.

Reply:
I do not think we should document the usage with `return_dag=True` anywhere, i.e., I would prefer to have #213 merged before #184. No need to rush in order to have it in the 0.3 release series.

Reply:
I agree. I will rewrite the interface here soon.